Formula/strace.rb

```diff
@@ -17,11 +17,14 @@ class Strace < Formula
   depends_on "linux-headers"
 
   def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --enable-mpers=check
```

Comment:
Why have you added this option `--enable-mpers=check`, and what does it do? Please add an explanation to the commit log message and pull request description.

Reply:
the default value is `yes` and formula fails to build. With `check` the build succeeds and I get the following:

```
checking for m32 personality compile support... yes
checking for m32 personality runtime support... no
```

Reply:
Please add a comment like `Disable m32 personality support`
And I'd suggest `--enable-mpers=no` for reproducible builds.
See https://github.com/strace/strace/blob/150655ca3e0f68f92dcb1e434301ac485aad5e15/configure.ac#L927

Reply:
Please post the build error message to this PR.

Reply:
You could add to the formula `# Fix ERROR-MESSAGE-GOES-HERE`

Reply:
posted log here: https://github.com/Linuxbrew/homebrew-extra/pull/38#issuecomment-413984644